include/libdnf/common/sack/query_cmp.hpp

```diff
@@ -122,24 +122,24 @@ enum class QueryCmp : uint32_t {
 };
 
 
-inline QueryCmp operator|(QueryCmp lhs, QueryCmp rhs) {
+inline constexpr QueryCmp operator|(QueryCmp lhs, QueryCmp rhs) noexcept {
     return static_cast<QueryCmp>(
         static_cast<std::underlying_type<QueryCmp>::type>(lhs) |
         static_cast<std::underlying_type<QueryCmp>::type>(rhs));
 }
 
 
-inline QueryCmp operator&(QueryCmp lhs, QueryCmp rhs) {
+inline constexpr QueryCmp operator&(QueryCmp lhs, QueryCmp rhs) noexcept {
     return static_cast<QueryCmp>(
         static_cast<std::underlying_type<QueryCmp>::type>(lhs) &
         static_cast<std::underlying_type<QueryCmp>::type>(rhs));
 }
 
 
-inline QueryCmp operator-(QueryCmp lhs, QueryCmp rhs) {
+inline constexpr QueryCmp operator-(QueryCmp lhs, QueryCmp rhs) noexcept {
```

Comment:
What about to add a dock string with short description what the operator does?

Reply:
OK.